### PR TITLE
First pass of aio transmitter

### PR DESCRIPTION
**Pull request by jondequinor**

### CODE REVIEW

File: ert3/data/_record.py [hunk 1]
```diff
@@ -5,12 +5,28 @@
...
+    async def run(*args, loop=None, executor=None, **kwargs):
+        if loop is None:
+            loop = asyncio.get_event_loop()
+        pfunc = partial(func, *args, **kwargs)
+        return await loop.run_in_executor(executor, pfunc)
+
+    return run
+
+
+_copy = wrap(shutil.copy)
```
Comment (sondreso) [inline ert3/data/_record.py:27]: I see we read the same issue 😄 
```suggestion
import aiofiles.os

_copy = aiofiles.os.wrap(shutil.copy)
```

**Review by sondreso — APPROVED** [review]

File: ert3/data/_record.py (hunk 1 above)
Comment (jondequinor) [reply ert3/data/_record.py:27]: 😎
Problem is that they don't export wrap

File: ert_shared/ensemble_evaluator/entity/ensemble.py [hunk 2]
```diff
@@ -35,8 +35,12 @@ def _sort_steps(steps: typing.List["_Step"]):
                 graph[other.get_name()].add(step.get_name())
                 edged_nodes.add(step.get_name())
                 edged_nodes.add(other.get_name())
+
     null_nodes = set([step.get_name() for step in steps]) - edged_nodes
-    [graph[node] for node in null_nodes]
+    for node in null_nodes:
+        # These null_nodes have no edges, so we just add them to the graph.
+        graph[node] = set()
+
```
Comment (sondreso) [inline ert_shared/ensemble_evaluator/entity/ensemble.py:43]: Is this a general bug that is not really related to the main focus of ths pr?

**Review by sondreso — APPROVED** [review]
